### PR TITLE
Apply small fixes and polish to vault / vert.x integration

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/client/MutinyVertxClientFactory.java [hunk 1]
```diff
@@ -19,8 +19,8 @@
     public static WebClient createHttpClient(Vertx vertx, VaultBootstrapConfig vaultBootstrapConfig, TlsConfig tlsConfig) {
 
         WebClientOptions options = new WebClientOptions()
-                .setConnectTimeout((int) vaultBootstrapConfig.connectTimeout.getSeconds())
-                .setIdleTimeout((int) vaultBootstrapConfig.readTimeout.getSeconds());
+                .setConnectTimeout((int) vaultBootstrapConfig.connectTimeout.toMillis())
```
Comment (geoand) [inline extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/client/MutinyVertxClientFactory.java:22]: The Javadoc of `ClientOptionsBase` says that the timeout is in milliseconds

Comment (vsevel) [reply extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/client/MutinyVertxClientFactory.java:22]: good catch for the connect timeout.
however, the idle timeout is in seconds:
```
Set the idle timeout, default time unit is seconds. Zero means don't timeout. This determines if a connection will timeout and be closed if no data is received within the timeout. If you want change default time unit, use setIdleTimeoutUnit(TimeUnit)
Params:
idleTimeout – the timeout, in seconds
Returns:
a reference to this, so the API can be used fluently
```

Comment (gsmet) [reply extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/client/MutinyVertxClientFactory.java:22]: That sounds about right :]

Comment (geoand) [reply extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/client/MutinyVertxClientFactory.java:22]: Wow, that is not intuitive...

Comment (geoand) [reply extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/client/MutinyVertxClientFactory.java:22]: Fixed

**Review by vsevel — APPROVED** [review]
I am glad you caught it. I am surprised actually that vault tests did not fail before. because the default connect timeout is 5 secs for the vault ext, which then was translated to 5 ms, which is no a lot of time to establish a connection...

**Comment by geoand** [comment]
I found it by accident when looking into making a similar switch for the Consul extension.
But it's a good thing that you then saw that connectionTimeout change was wrong so I can fix my thing as well ☺️

**Comment by geoand** [comment]
Ah sorry @vsevel , I hasn't seen you already approved, my bad :)

**Review by gsmet — APPROVED** [review]

**Pull request merged**